renderer/components/Settings/SettingsFieldHelpers.js

```diff
@@ -1,28 +1,5 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import { FormattedMessage } from 'react-intl'
-import { Box } from 'rebass/styled-components'
-import { Text } from 'components/UI'
-import { IntegerInput, Label } from 'components/Form'
-import messages from './messages'
-
-export const FieldLabel = ({ itemKey, ...rest }) => {
-  const messageKey = itemKey.replace('.', '_')
-  return (
-    <Box {...rest}>
-      <Label htmlFor={itemKey} mb={2}>
-        <FormattedMessage {...messages[`${messageKey}_label`]} />
-      </Label>
-      <Text color="gray" fontWeight="light">
-        <FormattedMessage {...messages[`${messageKey}_description`]} />
-      </Text>
-    </Box>
-  )
-}
-
-FieldLabel.propTypes = {
-  itemKey: PropTypes.string.isRequired,
-}
+import { IntegerInput } from 'components/Form'
 
 export const PositiveIntegerField = props => (
```

Comment:
Perhaps we should move `PositiveIntegerField` into `/components/Form`

Reply:
done